[2025-05-14T05:21:04.804+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_local_to_s3.upload_to_s3 manual__2025-05-14T05:21:04.482907+00:00 [queued]>
[2025-05-14T05:21:04.808+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_local_to_s3.upload_to_s3 manual__2025-05-14T05:21:04.482907+00:00 [queued]>
[2025-05-14T05:21:04.808+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-14T05:21:04.814+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2025-05-14 05:21:04.482907+00:00
[2025-05-14T05:21:04.817+0000] {standard_task_runner.py:57} INFO - Started process 302 to run task
[2025-05-14T05:21:04.821+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'upload_local_to_s3', 'upload_to_s3', 'manual__2025-05-14T05:21:04.482907+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/etl_redfin_to_s3.py', '--cfg-path', '/tmp/tmplnogbe7p']
[2025-05-14T05:21:04.824+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask upload_to_s3
[2025-05-14T05:21:04.849+0000] {task_command.py:416} INFO - Running <TaskInstance: upload_local_to_s3.upload_to_s3 manual__2025-05-14T05:21:04.482907+00:00 [running]> on host 23f1c69779e5
[2025-05-14T05:21:04.888+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_local_to_s3' AIRFLOW_CTX_TASK_ID='upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T05:21:04.482907+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-14T05:21:04.482907+00:00'
[2025-05-14T05:21:07.289+0000] {logging_mixin.py:154} INFO - ‚ùå Upload failed: Unable to locate credentials
[2025-05-14T05:21:07.289+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-14T05:21:07.294+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=upload_local_to_s3, task_id=upload_to_s3, execution_date=20250514T052104, start_date=20250514T052104, end_date=20250514T052107
[2025-05-14T05:21:07.340+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-14T05:21:07.348+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
